source/docs/transportation/alt-fuel-stations-v1/_request_params_common2.md.erb

```diff
@@ -411,7 +411,7 @@
       <strong>Default:</strong> <em>all</em>
     </div>
     <div class="doc-parameter-value-field">
-      <strong>Options:</strong> <em>all, AeroVironment Network, Blink Network, ChargePoint Network, EV Connect, eVgo Network, EVSE LLC WebNet, Greenlots, OpConnect, RechargeAccess, SemaCharge Network, Shorepower, Tesla</em>
+      <strong>Options:</strong> <em>all, AeroVironment Network, Blink Network, ChargePoint Network, EV Connect, eVgo Network, GE WattStation, Greenlots, OpConnect, SemaCharge Network, Tesla</em>
```

Comment:
Ah, gotcha, I see. I guess I might still be inclined to keep those the table values the same, since they're under the "Option" header in the table, where we typically list the coded values in the other tables. It's only under the "Description" headers where we typically show a "friendlier"/non-coded name (but in this case, the coded values are relatively readable, so that's why we hadn't bothered originally).

In any case, it also looks like there's still 1 `EVSE LLC WebNet` network station, so should we have removed that? Or is that maybe a data issue for ICF to address if that network is no longer supposed to be active?

This API gives a more definitive list of all the EVSE networks that are active in our system (which we should really document, so maybe we wouldn't have to maintain this static list, but that's probably another issue): http://developer.nrel.gov/api/alt-fuel-stations/v1/electric-networks.json?api_key=DEMO_KEY So here's the 1 station that's still a part of this network: http://www.afdc.energy.gov/locator/stations/places/64957